src/components/startups/Welcome.vue

```diff
@@ -306,6 +306,11 @@ export default {
                 }
             });
 
+            // switch to server buffer if no channels are joined
+            if (!hasSwitchedActiveBuffer) {
+                state.setActiveBuffer(net.id, '*');
```

Comment:
`net.serverBuffer().name` would be preferred over `*` here for readability

Reply:
forced pushed, thanks